app/src/app/components/specmath-modal/modal.component.spec.ts

```diff
@@ -15,29 +15,26 @@
 import { TestBed, async, ComponentFixture } from '@angular/core/testing';
 import { ModalComponent } from './modal.component';
 
-import { MatDialogModule, MatDialog, MatDialogRef } from '@angular/material/dialog';
-import { By, BrowserModule } from '@angular/platform-browser';
+import { MatDialogModule, MatDialogRef } from '@angular/material/dialog';
+import { BrowserModule } from '@angular/platform-browser';
 import { MatStepperModule } from '@angular/material/stepper';
 import { MatButtonModule } from '@angular/material/button';
 import { MatIconModule } from '@angular/material/icon';
 import { MatInputModule } from '@angular/material/input';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { MatTooltipModule } from '@angular/material/tooltip';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-
-const queryElement = (targetFixture: ComponentFixture<ModalComponent>, targetClass: string) => {
-  return targetFixture.debugElement.query(By.css(targetClass));
-};
+import { Step1Component } from './specmath-modal-step-1/step1.component';
+import { queryElement } from '../../../shared/functions';
 
 describe('ModalComponent', () => {
-  let dialog: MatDialog;
   let fixture: ComponentFixture<ModalComponent>;
   let modal: ModalComponent;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       declarations: [
-        ModalComponent
+        ModalComponent, Step1Component
```

Comment:
I recommend, where possible, stubbing components referenced by the component under test...